tasks/build-tachometer-comment.js

```diff
@@ -20,7 +20,7 @@ const getTachometerResults = () => {
         chrome: [],
         firefox: [],
     };
-    for (const result of fg.sync(`./tach-results.*.json`)) {
+    for (const result of fg.sync(`./*/tach-results.*.json`)) {
```

Comment:
Good catch here - we chatted in slack about updating this to a more specific path: `./tachometer-results-*/tach-results.*.json`